addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -1389,6 +1389,15 @@ export function getOuid(node, optimize = false) {
     }
     return node && node.oid;
 }
+/**
+ * Returns true if the provided node can suport html content
+ *
+ * @param {Node} node
+ * @returns {boolean}
+ */
+export function nodeSupportHtmlContent(node) {
```

Comment:
To be coherent with the naming scheme of the other utils, I suggest `isHtmlContentSupported`, what do you think ?